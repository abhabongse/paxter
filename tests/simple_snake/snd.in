@!{
x = 1
def add(_, a, b):
    return a + b
}
@x and @add[a=x,b=2]{}
