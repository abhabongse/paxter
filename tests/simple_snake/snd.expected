
1 and 3
